src/kakarot/accounts/account_contract.cairo

```diff
@@ -139,6 +211,9 @@ func __execute__{
     response_len: felt, response: felt*
 ) {
     alloc_locals;
+    with_attr error_message("EOA: __execute__ not supported") {
+        assert 1 = 0;
+    }
```

Comment:
remove all code after

Reply:
and no need to `alloc_locals` anymore (though it changes nothing for the actual run)

Reply:
Done with [0f8d8c4](https://github.com/kkrt-labs/kakarot/pull/1246/commits/0f8d8c4e9e8e8e6a80028b5b5496bc8649d528a6) and [ae7abd4](https://github.com/kkrt-labs/kakarot/pull/1246/commits/ae7abd4635fa5b09fd19179339c099b5f0e9678b)